chain/epoch_manager/src/lib.rs

```diff
@@ -354,7 +354,7 @@ impl EpochManager {
         block_info: &BlockInfo,
         last_block_hash: &CryptoHash,
         rng_seed: RngSeed,
-    ) -> Result<EpochId, EpochError> {
+    ) -> Result<(), EpochError> {
```

Comment:
What is this change for?

Reply:
I don't see the return value used anywhere